### PR TITLE
converting all normal functions to arrow functions in utils

### LINKED ISSUE #1683: Refactoring in ./src/utils
**Actual Behaviour**

The current functions in utils use normal JS functions.

**Expected Behaviour**

As we plan to have a more consistent code base, it should be replaced with arrow functions.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #1683 

Changes: I have converted all the normal functions to arrow functions.

Demo Link: https://pr-1684-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -1,2 +1,4 @@
-export { default as camelizeKeys } from './camelizeKeys';
-export { default as toFormData } from './toFormData';
+import { camelizeKeys } from './camelizeKeys';
+import { toFormData } from './toFormData';
+
```
Comment (akshatnitd) [inline src/utils/index.js]: Was change required in this file?

Comment (AakashMallik) [reply src/utils/index.js]: Sir, I thought it would be more consistent to the rest of the codebase. Should I revert the change?

Comment (akshatnitd) [reply src/utils/index.js]: Yes, I think you should revert the changes of the file. The current approach of exporting is better,

**Comment by AakashMallik** [comment]
I have made the changes you asked for, sir.

**Comment by AakashMallik** [comment]
@akshatnitd Yes sir, I have changed my commit message to fit the template.

**Pull request merged**